src/main/java/org/example/service/DeveloperService.java

```diff
@@ -0,0 +1,55 @@
+package org.example.service;
+
+import org.example.model.Developer;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeveloperService {
+
+    private static final String GET_DEVELOPERS = "select user_id, firstname, lastname, age, proglang from developer";
+
+    private final ConnectionService connectionService = ConnectionService.getInstance();
+
+    private static DeveloperService INSTANCE = null;
+
+    public List<Developer> getDevelopers() {
+        Connection connection = connectionService.getConnection();
+        List<Developer> developers = new ArrayList<>();
+        try (
+                Statement statement = connection.createStatement();
+                ResultSet resultSet = statement.executeQuery(GET_DEVELOPERS)
+        ) {
+            while (resultSet.next()) {
+                int userId = resultSet.getInt("user_id");
+                String firstname = resultSet.getString("firstname");
+                String lastname = resultSet.getString("lastname");
+                int age = resultSet.getInt("age");
+                String proglang = resultSet.getString("proglang");
+
+                Developer developer = new Developer();
+                developer.setUserId(userId);
+                developer.setFirstName(firstname);
+                developer.setSecondName(lastname);
+                developer.setAge(age);
+                developer.setProgLang(proglang);
+
+                developers.add(developer);
+            }
+        } catch (SQLException ex) {
+            System.out.println(ex);
```

Comment:
Check the best practices to handle exceptions in Java.

Reply:
I read about 'try', 'catch' and 'finally'. In catch you write a message which describe problem of exception more deeply. Then you put finally to clean up resources. Is that what you want in this issue ?

Reply:
Yes, right.